docs/0_prerequisites.md

```diff
@@ -7,9 +7,12 @@ To use ReVanced CLI, you will need to fulfill certain requirements.
 - Java SDK 11 (Azul JDK or OpenJDK)
 - [Android Debug Bridge (adb)](https://developer.android.com/studio/command-line/adb) if you want to deploy the patched APK file on your device
 - An ABI other than ARMv7 such as x86 or x86-64 (or a custom AAPT binary that supports ARMv7)
+- Patches such as [Revanced Patches](https://github.com/ReVanced/revanced-patches/releases)
 
-## ⏭️ Whats next
+## 🔹 Optional
```

Comment:
`## 🔹 Optional` is at the same level as `## 🤝 Requirements` but should be semantically under `## 🤝 Requirements` as an "optional" requirement.

Regardless, integrations are not required to use ReVanced CLI at any stage, including patching. The usage page mentions that some patches may require integrations and how you can supply them.